flight/targets/colibri/fw/pios_board.c

```diff
@@ -692,7 +692,7 @@ void PIOS_Board_Init(void)
 		}
 
 		if (PIOS_I2C_CheckClear(pios_i2c_usart1_adapter_id) != 0)
-			panic(6);
+			AlarmsSet(SYSTEMALARMS_ALARM_I2C, SYSTEMALARMS_ALARM_CRITICAL);
```

Comment:
Checking the I2C is clear should still be a panic. If nothing is there the bus should still panic. This should be fixed throughout.